Mathlib/LinearAlgebra/TensorProduct/Basic.lean

```diff
@@ -757,6 +757,14 @@ theorem map_comp (f₂ : P →ₗ[R] P') (f₁ : M →ₗ[R] P) (g₂ : Q →ₗ
     map (f₂.comp f₁) (g₂.comp g₁) = (map f₂ g₂).comp (map f₁ g₁) :=
   ext' fun _ _ => rfl
 
+lemma map_map {M₁ M₂ M₃ N₁ N₂ N₃ : Type*}
+    [AddCommMonoid M₁] [Module R M₁] [AddCommMonoid N₁] [Module R N₁]
+    [AddCommMonoid M₂] [Module R M₂] [AddCommMonoid N₂] [Module R N₂]
+    [AddCommMonoid M₃] [Module R M₃] [AddCommMonoid N₃] [Module R N₃]
+    (f₂ : M₂ →ₗ[R] M₃) (g₂ : N₂ →ₗ[R] N₃) (f₁ : M₁ →ₗ[R] M₂) (g₁ : N₁ →ₗ[R] N₂) (x : M₁ ⊗ N₁) :
+    map f₂ g₂ (map f₁ g₁ x) = map (f₂ ∘ₗ f₁) (g₂ ∘ₗ g₁) x :=
+  DFunLike.congr_fun (map_comp ..).symm x
```

Comment:
Can't you reuse the variables from `map_comp` above?

Reply:
(or if you feel strongly that these are better, replace the variables in `map_comp` to match)

Reply:
Good point